addons/website/static/src/xml/website.res_config_settings.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
```

Comment:
If you agree, I would unindent the first level of all your static files

Reply:
Yes, I prefer that way too, it's just that lot of `/views/` files are doing it while most `/static/` don't so I kept the same stuff.